src/components/cards/DataSummaryCardGroup.less

```diff
@@ -0,0 +1,51 @@
+@import '../../css/base.less';
+
+.data-summary-card-group {
+  display: flex;
+}
+  
+.data-summary-card-group__connected-card,
+.data-summary-card-group__separated-card {
+  background-color: #ffffff;
+  flex-grow: 1;
+  flex-basis: 0;
+  position: relative;
+}
+
+.data-summary-card-group__separated-card {
+  margin-left: 15px;
+  &:first-child {
+    margin-left: 0px;
+  }
+}
+
+.data-summary-card-group__connected-card {
+  margin: 0;
+  &:first-child {
+    border: none;
+  }
+
+  .left-border {
+    position: absolute;
+    top: 7px;
+    bottom: 7px;
+    border-left: solid @silver 1px;
+  }
+}
+
+.data-summary-card-group__sub-card-group {
+  display: flex;
+  height: 100%;
+}
+
+.data-summary-card-group__sub-card-item {
+  flex-grow: 1;
+  height: 100%;
+
+  .left-border {
```

Comment:
same here

Reply:
Just thought nesting left-border is more semantic and easier to read...... 😛anyways, I can change to the top level and we now stick with BEM...